src/dailies.ts

```diff
@@ -541,3 +541,13 @@ export function gingerbreadPrepNoon(): void {
     }
   }
 }
+
+export function martini(): void {
+  if (
+    !have($item`Kremlin's Greatest Briefcase`) ||
+    get("_kgbClicksUsed") > 17 ||
+    get("_kgbDispenserUses") >= 3
+  )
+    return;
```

Comment:
is it worth putting {} around the return, just to avoid confusion? I had to parse it a couple of times before the segmentation stuck.